examples/auto-relay/README.md

```diff
@@ -106,18 +106,15 @@ console.log(`Node started with id ${node.peerId.toB58String()}`)
 
 const conn = await node.dial(relayAddr)
 
+console.log(`Connected to the HOP relay ${conn.remotePeer.toString()}`)
+
 // Wait for connection and relay to be bind for the example purpose
-await new Promise((resolve) => {
-  node.peerStore.on('change:multiaddrs', ({ peerId }) => {
-    // Updated self multiaddrs?
-    if (peerId.equals(node.peerId)) {
-      resolve()
-    }
-  })
+node.peerStore.on('change:multiaddrs', ({ peerId }) => {
```

Comment:
The things here is that if your node stops, you want wait for this... Can we keep the HOP relay info?

Reply:
You mean the running node? What do you mean by HOP relay? Can you help me out?

Reply:
anyway what was HOP relay can you help me find it in docs? :D